document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the  function being         | values, in which case,      |                             |
+|                  | returned from.              | string representations      |                             |
+|                  |                             | of the value or values      |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|                  |                             |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `continuation`   | Absent                      | Absent                      | Optional: The names and     |
+|                  |                             |                             | values of selected variables|
+|                  |                             |                             | at the specified location.  |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `declaration`    | Recommended: The            | Optional: If the            | Optional: If the            |
+|                  | name of the variable        | declaration has an          | declaration has an          |
+|                  | being declared. If          | initializer, in which       | initializer, in which case, |
+|                  | multiple variables are      | case, the value of          | the names and values of     |
+|                  | declared in the same        | the initializer             | any variables participating |
+|                  | statement, a separate       | expression.                 | in the initializer          |
+|                  | `annotatedCodeLocation`     | Absent: If the              | expression.                 |
+|                  | object shall be created     | declaration has no          | Absent: If the              |
+|                  | for each declaration that   | initializer.                | declaration has no          |
+|                  | the tool creating the       |                             | initializer.                |
+|                  | log wishes to represent.    |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionEnter`  | Recommended: The            | Absent                      | Absent                      |
```

Comment:
>Absent [](start = 51, length = 6)

if we provide for return values in functionExit, why would we not permit parameterization on functionEnter? Probably should add it. It it fine to mark these as optional.